### ISSUE TITLE
sourcemap is likely to be incorrect PART II with example repo

### ISSUE THREAD

**Issue by thisconnect** [rollup/rollup-plugin-babel#278]
```
git clone https://github.com/thisconnect/test-rollup-babel-sourcemap
cd test-rollup-babel-sourcemap
npm i
npm start
```

output 

```
Sourcemap is likely to be incorrect: a plugin ('babel') was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

**Comment by thisconnect** [comment]
https://github.com/thisconnect/test-rollup-babel-sourcemap

**Comment by Andarist** [comment]
I believe I have fixed this. Could you try `rollup-plugin-babel@4.1.0-1`?

**Comment by thisconnect** [comment]
> I believe I have fixed this. Could you try rollup-plugin-babel@4.1.0-1?

👍 perfect, is `rollup-plugin-babel@4.1.0-1` a prerelease or do you suggest to use this version in production?

**Comment by Andarist** [comment]
It's a prerelease so you can easily test it but as soon as you confirm that it works I'm going to release it as latest.

**Comment by thisconnect** [comment]
I confirm, it is fixed with 4.1.0-1 
updated test repo 
https://github.com/thisconnect/test-rollup-babel-sourcemap/commit/2d984f5defde65441013c1ad43a0bdeedc39072e

**Comment by Andarist** [comment]
Published as 4.1.0
